DocumentService/Repositories/DocumentRepository.cs

```diff
@@ -142,6 +162,57 @@ private async Task<DocumentInfo> GetDocument(Guid id)
             return await query.FirstOrDefaultAsync();
 
         }
+        private DateTime GetTimeForNCR()
+        {
+            try
+            {
+                DateTime timeUTC = DateTime.UtcNow;
+                TimeZoneInfo estZone = TimeZoneInfo.FindSystemTimeZoneById("Eastern Standard Time");
+                DateTime estTime = TimeZoneInfo.ConvertTimeFromUtc(timeUTC, estZone);
+                return estTime;
+            }
+            catch (TimeZoneNotFoundException)
+            {
+                throw;
+            }
+            
+        }
+        private List<DocumentInfo> PopulateDocumentInfo(DocumentDTO documentDTO, HttpContext httpContext)
```

Comment:
Same here remove the HttpContext.
```suggestion
        private List<DocumentInfo> PopulateDocumentInfo(DocumentDTO documentDTO)
```